components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/GatewaySynapseArtifactsApiService.java

```diff
@@ -0,0 +1,25 @@
+package org.wso2.carbon.apimgt.internal.service;
+
+import org.wso2.carbon.apimgt.internal.service.*;
```

Comment:
Import the exact Class rather than using wildcard.

Reply:
This is an auto-generated file. We don't need to modify this.